python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py

```diff
@@ -36,6 +35,8 @@
 import zipfile as _zipfile
 from glob import glob as _glob
 
+import math as _math
```

Comment:
Why have you changed the order?

Reply:
pycharm did it for me, let me recreate the env and run it again.